services/ymax-planner/src/plan-deposit.ts

```diff
@@ -212,8 +212,12 @@ export const getCurrentBalances = async (
       makeSpectrumPoolQuery(desc, powers),
     );
     const [accountResult, positionResult] = await Promise.allSettled([
-      spectrumBlockchain.getBalances({ accounts: spectrumAccountQueries }),
-      spectrumPools.getBalances({ positions: spectrumPoolQueries }),
+      spectrumAccountQueries.length
```

Comment:
This is observable at runtime and so doesn't fit my notion of "refactor".

I'm curious where you draw the line.

not critical. the commit is small enough that it doesn't matter which lens I review it through.